packages/server/graphql/types/TeamPromptResponse.ts

```diff
@@ -0,0 +1,55 @@
+import {GraphQLFloat, GraphQLID, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import Team from './Team'
+
+const TeamPromptResponse: GraphQLObjectType = new GraphQLObjectType<any, GQLContext>({
+  name: 'TeamPromptResponse',
+  description: 'A response of a single team member in a team prompt',
+  isTypeOf: ({status}) => !!status,
+  fields: () => ({
+    createdBy: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The userId that created the item'
+    },
+    createdByUser: {
+      type: new GraphQLNonNull(require('./User').default),
+      description: 'The user that created the item',
+      resolve: ({createdBy}, _args: unknown, {dataLoader}: GQLContext) => {
```

Comment:
-2 In DB, the field is named `userId`

Reply:
this whole thing wasn't really needed, we already have `user` and `userId` which is enough.